test/integration/guest_user_test.rb

```diff
@@ -6,9 +6,9 @@ class GuestUserTest < ActionDispatch::IntegrationTest
   attr_reader :item1, :item2, :category1, :category2
 
   def setup
-    @item1 = Item.create(name: 'espresso', cost: 9000)
+    @item1 = Item.create(title: 'espresso', price: 9000)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.